ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -441,7 +438,11 @@ objects:
         wait "$!"
         install_exit_status=$?
         sleep 10m
-        oc get co/image-registry
+        # This is a workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1794714
+        for n in $(oc --insecure-skip-tls-verify get nodes|awk '{print $1}'|grep worker);do
+          oc -n default --insecure-skip-tls-verify debug node/$n --image=centos/tools -- ethtool --offload vxlan_sys_4789 tx off
```

Comment:
I feel like this is the sort of change that should be handled by [a MachineConfig][1], although I don't know what you'd put in the config to apply this specific `ethtool` tweak.  Dropping into a debug session on each node and bumping things yourself seems very brittle.

[1]: https://github.com/openshift/machine-config-operator/blob/master/docs/MachineConfiguration.md

Reply:
When we talked about it we said that we don't want to apply it by default on a cluster because it is a tweak that disables checksum and it seems to us like a decision that we don't want by default, plus it is a high priority bug which affects most platforms, happens only when using OpenshiftSDN , and has network people working on it.
We wanted to apply the fix in the CI because it made network test flaky during conformance.